### PR TITLE
Formatted callback.py to render correctly

### PR DESCRIPTION
**Pull request by jvishnuvardhan**
Updated some formatting to render the code block correctly

### CODE REVIEW

**Comment by rchao** [comment]
Thanks for the PR! The code change will be problematic because it's using doctest which attempts to run the example code, but it's actually not runnable. Can you do the couple things instead: 1) add 'python' after the initial '```' and 2) fix the indentation.

**Review by fchollet — APPROVED** [review]
Thanks for the update. LGTM.

**Comment by fchollet** [comment]
Please fix the code formatting:

```
keras/callbacks.py:643:1: W293 blank line contains whitespace
keras/callbacks.py:[6](https://github.com/keras-team/keras/runs/7238487569?check_suite_focus=true#step:6:7)59:1: W293 blank line contains whitespace
```

**Comment by jvishnuvardhan** [comment]
> Please fix the code formatting:
> 
> ```
> keras/callbacks.py:643:1: W293 blank line contains whitespace
> keras/callbacks.py:[6](https://github.com/keras-team/keras/runs/7238487569?check_suite_focus=true#step:6:7)59:1: W293 blank line contains whitespace
> ```

Updated. Please review. Thanks

**Review by fchollet — APPROVED** [review]

File: keras/callbacks.py [hunk 1]
```diff
@@ -654,7 +654,8 @@ class Callback:
          callbacks.on_epoch_end(epoch, epoch_logs)
        final_logs=...
        callbacks.on_train_end(final_logs)
-       ```
+    ```
+
```
Comment (fchollet) [inline keras/callbacks.py:658]: Remove whitespace.

**Review by fchollet — COMMENTED** [review]
Please fix the `black` formatting test.

**Review by fchollet — APPROVED** [review]

**Pull request merged**